aten/src/ATen/native/mps/operations/Unique.mm

```diff
@@ -319,13 +319,13 @@ static void runUniqueGraph(UniqueCachedGraph* uniqueGraph,
 static Tensor lexsort_rows_perm_mps(const Tensor& mat_2d) {
   const auto rows = mat_2d.size(0), cols = mat_2d.size(1);
   if (rows <= 1 || cols == 0) {
-    return arange(rows, mat_2d.options().dtype(kLong));
+    return at::arange(rows, mat_2d.options().dtype(kLong));
```

Comment:
Those changes are not wrong, but unrelated to PR title, may be could be submitted as part of separate PR?